ut_core.py

```diff
@@ -0,0 +1,184 @@
+
+import sys
+import yaml
+import re
+
+from framework.core.logModule import logModule
+
+class UTCoreMenuNavigator:
```

Comment:
need a document on the usage of this module with maybe the format requirements and or also an example of it added to the examples.

Reply:
is it something like this?

```yaml
# this file is generated by the testing suites, when a new test is modified, this configuration file will be filled out and updated.

dsAudio:  # Prefix must always exist
  description: "dsAudio Device Settings testing profile / menu system for UT"
  test:
  control:
    menu:
      type: UT-C # C (UT-C Cunit) / C++ (UT-G (g++ ut-core gtest backend))
      groups: 
          name: "L3 dsAudio - Sink"
          menu_initialize:
            name: "Initialize dsAudio"
          menu_enable:
            name: "Enable Audio Port"
            input: {"Select dsAudio Port": "dsAUDIOPORT_TYPE_SPEAKER"}
          menu_disable:
            name: "Disable Audio Port"
            input: {"Select dsAudio Port": "dsAUDIOPORT_TYPE_SPEAKER"}
          menu_terminate:
            name: "Terminate dsAudio"
```

Reply:
zghp seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

Something wrong with the commit history of this branch, it looks like it requires fixing.

Reply:
should ut-helper needs to inherit this class also, so that the helper has it, it will need the navigation through the menus for sure.